### PR TITLE
Adding in tee of run to be able to pass output to write to sheet

### PR DESCRIPTION
**Pull request by paigerube14**
Using tee to print output in real time and be able to pass file to write to sheet. Also adding passing of variables from the specific run type. 

Taking out extra folders that were copied over from main branch

This needs to be tested/merged after https://github.com/openshift-qe/ocp-qe-perfscale-ci/pull/107 goes in

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -149,15 +152,19 @@ pipeline {
                 currentBuild.result = "FAILURE"
             }
            if(params.WRITE_TO_FILE == true) {
-            build job: 'scale-ci/e2e-benchmarking-multibranch-pipeline/write-scale-ci-results', parameters: [string(name: 'BUILD_NUMBER', value: BUILD_NUMBER),text(name: "ENV_VARS", value: ENV_VARS),string(name: 'CI_JOB_ID', value: BUILD_ID), string(name: 'CI_JOB_URL', value: BUILD_URL), string(name: 'JENKINS_AGENT_LABEL', value: JENKINS_AGENT_LABEL), string(name: "CI_STATUS", value: "${status}"), string(name: "JOB", value: WORKLOAD)]
+
+            def parameter_to_pass = VARIABLE
+            if (params.WORKLOAD == "node-density" || params.CI_TYPE == "node-density-heavy" ) {
```
Comment (qiliRedHat) [inline Jenkinsfile:157]: CI_TYPE is supposed to be WORKLOAD?

Comment (paigerube14) [reply Jenkinsfile:157]: yes, missed that in copying thanks!! Fixing now

**Pull request merged**